src/QuantumSdk/DefaultItems/DefaultItems.targets

```diff
@@ -59,8 +59,9 @@
   <PropertyGroup>
     <DefaultEntryPointGeneration>dotnet "$(MSBuildThisFileDirectory)../tools/utils/Microsoft.Quantum.Sdk.DefaultEntryPoint.Generation.dll"</DefaultEntryPointGeneration>
     <!-- true/false optional properties -->
-    <QirGeneration Condition="'$(QirOutputPath)' != '' And '$(QirGeneration)' == 'default'">true</QirGeneration>
-    <QirGeneration Condition="'$(QirOutputPath)' == '' And '$(QirGeneration)' == 'default'">false</QirGeneration>
+    <Warning Condition="'$(QirGeneration)' == 'true' And '$(ResolvedQSharpOutputType)' == 'QSharpLibrary'" Text="QIR Generation 'true' is not supported on library projects, returning to default behavior." />
```

Comment:
Looks like the comparison of `$QirGeneration` with 'true' is causing an issue when it is expecting `$QirGeneration` to have the value 'default'.

Reply:
Ah, you're right, this is causing build failures here. I can get that fixed and try again.